storage/inspector/src/main.rs

```diff
@@ -120,6 +124,36 @@ fn list_txns(db: &LibraDB) {
     }
 }
 
+fn list_accounts(db: &LibraDB) {
+    let version = db
+        .get_latest_version()
+        .expect("Unable to get latest version");
+    let backup = db.get_backup_handler();
+    let iter = backup
+        .get_account_iter(version)
+        .expect("Unagle to get account iter");
+    for v in iter {
+        match v {
+            Ok(x) => {
+                let accs = AccountState::try_from(&x.1).expect("Failed to read AccountState");
+                let addr = accs
+                    .get_account_address()
+                    .expect("Could not get address from state");
+                match addr {
+                    Some(x) => println!("Address: {:?}", x),
+                    None => println!("Invalid Address: {:?}", addr),
```

Comment:
Here you are just printing "None" ,  which you should say  "no address found for blob X, skipped" something like this.